templates/mautic/0/rancher-compose.yml

```diff
@@ -0,0 +1,71 @@
+version: '2'
+
+catalog:
+  name: "Mautic"
+  version: "v2.12"
+  description: "Mautic provides free and open source marketing automation software available to everyone. Free email marketing software, lead management software and more."
+  uuid: Mautic-0
+  minimum_rancher_version: v0.51.0
+  questions:
+    - variable: mautic_version
+      description: "Mautic version/Docker tag used for the Mautic container images."
+      label: "MAUTIC_VERSION"
+      type: enum
+      default: "2.12-apache"
+      options:
+        - 2.12-apache
+        - 2.11-apache
+        - 2.10-apache
+        - 2.9-apache
+        - 2.8-apache
```

Comment:
Better to maintain deployed version with catalog package. Here may be make sense to be able to select between apache or fpm version. :)

Reply:
I like this idea unless we stick to fpm and add nginx in front of it for better load balancing/performance